tests/parallel_testsuite.py

```diff
@@ -48,8 +50,14 @@ def __init__(self, max_cores):
 
   def run(self, result):
     test_queue = self.create_test_queue()
-    self.init_processes(test_queue)
-    results = self.collect_results()
+    try:
+      self.init_processes(test_queue)
```

Comment:
does this line need to be inside the try?

Reply:
Presumably if init-ing fails we might need to clean up... forget what it does and if/how it can fail though.